### PR TITLE
[DEV-11239] Implement ButtonBlockNode renderer in `@prezly/content-renderer-react-js`

### PR DESCRIPTION
**Pull request by fgyimah**

### CODE REVIEW

File: src/elements/ButtonBlock/ButtonBlock.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    node: ButtonBlockNode;
+}
+
+export function ButtonBlock({ node }: ButtonNodeProps) {
+    const { href, label, layout, new_tab: newTab, variant } = node;
+
+    return (
+        <a
+            className={classNames('prezly-slate-button-block', {
+                'prezly-slate-button-block--disabled': !href,
```
Comment (e1himself) [inline src/elements/ButtonBlock/ButtonBlock.tsx:16]: We don't want to render buttons without `href`.
See https://linear.app/prezly/issue/DEV-11229/design-buttons-in-storyless-campaigns#comment-701415b7

File: src/elements/ButtonBlock/ButtonBlock.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+                'prezly-slate-button-block--outlined': variant === ButtonBlockNode.Variant.OUTLINE,
+                'prezly-slate-button-block--fullWidth': layout === ButtonBlockNode.Layout.WIDE,
```
Comment (e1himself) [inline src/elements/ButtonBlock/ButtonBlock.tsx:18]: Let's match the class name with the layout name:

```suggestion
                'prezly-slate-button-block--wide': layout === ButtonBlockNode.Layout.WIDE,
```

**Review by e1himself — CHANGES_REQUESTED** [review]
Looks good.

Two small comments below. Otherwise good :+1:

File: src/elements/ButtonBlock/ButtonBlock.tsx (hunk 1 above)
Comment (fgyimah) [reply src/elements/ButtonBlock/ButtonBlock.tsx]: Updated

**Review by e1himself — APPROVED** [review]

**Pull request merged**